Merge FDMI patches to fdmi_int

huanghua78 opened the pull request:
The following PRs:
#736 
#737 
#774 
#777

shailesh-vaidya commented:
<h1>Jenkins CI Result : <a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645/>Motr#645</a></h1><h2>Motr Test Summary</h2><table><tr><th>Test Result</th><th>Count</th><th>Info</tr><tr><td>:x:Failed</td><td>2</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/51kem target=_blank>02motr-single-node/51kem</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/35m0singlenode target=_blank>01motr-single-node/35m0singlenode</a></p></details></td></tr><tr><td>:checkered_flag:Skipped</td><td>5</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/37protocol target=_blank>01motr-single-node/37protocol</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/31motr-sys-kvs-kernel-test target=_blank>04motr-single-node/31motr-sys-kvs-kernel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/07multi-client-tests target=_blank>05motr-single-node/07multi-client-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/12sns-cc-io-tests target=_blank>05motr-single-node/12sns-cc-io-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/39mount-fail target=_blank>05motr-single-node/39mount-fail</a></p></details></td></tr><tr><td>:heavy_check_mark:Passed</td><td>57</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/00userspace-tests target=_blank>01motr-single-node/00userspace-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/01kernel-tests target=_blank>01motr-single-node/01kernel-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/03initscript-tests target=_blank>01motr-single-node/03initscript-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/04net-tests target=_blank>01motr-single-node/04net-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/05rpcping-tests target=_blank>01motr-single-node/05rpcping-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/06console-tests target=_blank>01motr-single-node/06console-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/28motr-sys-kvs-test target=_blank>01motr-single-node/28motr-sys-kvs-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/29confgen-test target=_blank>01motr-single-node/29confgen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/30hagen-test target=_blank>01motr-single-node/30hagen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/46m0crate target=_blank>01motr-single-node/46m0crate</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/01motr-single-node/52singlenode-sanity target=_blank>01motr-single-node/52singlenode-sanity</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/02system-tests target=_blank>02motr-single-node/02system-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/08degraded-mode-tests target=_blank>02motr-single-node/08degraded-mode-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/09poolmach-tests target=_blank>02motr-single-node/09poolmach-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/20pool-version-test target=_blank>02motr-single-node/20pool-version-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/21spiel-test target=_blank>02motr-single-node/21spiel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/22rpc-cancel-test target=_blank>02motr-single-node/22rpc-cancel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/23m0d-signal-test target=_blank>02motr-single-node/23m0d-signal-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/24m0d-fsync-test target=_blank>02motr-single-node/24m0d-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/41motr-conf-update target=_blank>02motr-single-node/41motr-conf-update</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/47motr-tests-user-kernel target=_blank>02motr-single-node/47motr-tests-user-kernel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/02motr-single-node/53clusterusage-alert target=_blank>02motr-single-node/53clusterusage-alert</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/16conf-st-test target=_blank>03motr-single-node/16conf-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/17sss-st-test target=_blank>03motr-single-node/17sss-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/36spare-reservation target=_blank>03motr-single-node/36spare-reservation</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/54sns-repair-motr-1f target=_blank>03motr-single-node/54sns-repair-motr-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/55sns-repair-motr-1n-1f target=_blank>03motr-single-node/55sns-repair-motr-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/56sns-repair-motr-mf target=_blank>03motr-single-node/56sns-repair-motr-mf</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/57sns-repair-motr-1k-1f target=_blank>03motr-single-node/57sns-repair-motr-1k-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/58sns-repair-motr-ios-fail target=_blank>03motr-single-node/58sns-repair-motr-ios-fail</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/59sns-repair-motr-abort target=_blank>03motr-single-node/59sns-repair-motr-abort</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/60sns-repair-motr-abort-quiesce target=_blank>03motr-single-node/60sns-repair-motr-abort-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/03motr-single-node/61spiel-multi-confd target=_blank>03motr-single-node/61spiel-multi-confd</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/19sns-abort-test target=_blank>04motr-single-node/19sns-abort-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/25spiel-sns-repair-test target=_blank>04motr-single-node/25spiel-sns-repair-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/26spiel-sns-repair-quiesce-test target=_blank>04motr-single-node/26spiel-sns-repair-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/27sns-repair-io-fail-test target=_blank>04motr-single-node/27sns-repair-io-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/32m0t1fs-rconfc-fail-test target=_blank>04motr-single-node/32m0t1fs-rconfc-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/33motr-st target=_blank>04motr-single-node/33motr-st</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/34sns-repair-1n-1f target=_blank>04motr-single-node/34sns-repair-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/48motr-raid0-io target=_blank>04motr-single-node/48motr-raid0-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/49motr-rpc-cancel target=_blank>04motr-single-node/49motr-rpc-cancel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/50motr-rm-lock-cc-io target=_blank>04motr-single-node/50motr-rm-lock-cc-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/04motr-single-node/51motr-rmw target=_blank>04motr-single-node/51motr-rmw</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/10sns-single-tests target=_blank>05motr-single-node/10sns-single-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/11sns-multi-tests target=_blank>05motr-single-node/11sns-multi-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/13sns-repair-quiesce target=_blank>05motr-single-node/13sns-repair-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/14m0t1fs-fsync-test target=_blank>05motr-single-node/14m0t1fs-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/15m0t1fs-fwait-test target=_blank>05motr-single-node/15m0t1fs-fwait-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/18m0mount-test target=_blank>05motr-single-node/18m0mount-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/38sns-abort-quiesce-test target=_blank>05motr-single-node/38sns-abort-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/40motr-dgmode target=_blank>05motr-single-node/40motr-dgmode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/42motr-utils target=_blank>05motr-single-node/42motr-utils</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/43motr-sync-replication target=_blank>05motr-single-node/43motr-sync-replication</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/44motr-sns-repair target=_blank>05motr-single-node/44motr-sns-repair</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/05motr-single-node/45motr-sns-repair-N-1 target=_blank>05motr-single-node/45motr-sns-repair-N-1</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit/motr-single-node/62dix-repair-st-lookup-insert target=_blank>motr-single-node/62dix-repair-st-lookup-insert</a></p></details></td></tr><tr><td>Total</td><td>64</td><td><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/645//testReport/junit>:link:</a></td></tr></table><h2> CppCheck Summary</h2><p> &emsp;&emsp;&emsp;Cppcheck: No new warnings found :thumbsup:</p>

shailesh-vaidya commented:
<h1>Jenkins CI Result : <a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646/>Motr#646</a></h1><h2>Motr Test Summary</h2><table><tr><th>Test Result</th><th>Count</th><th>Info</tr><tr><td>:x:Failed</td><td>2</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/49motr-rpc-cancel target=_blank>04motr-single-node/49motr-rpc-cancel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/51kem target=_blank>02motr-single-node/51kem</a></p></details></td></tr><tr><td>:checkered_flag:Skipped</td><td>5</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/37protocol target=_blank>01motr-single-node/37protocol</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/31motr-sys-kvs-kernel-test target=_blank>04motr-single-node/31motr-sys-kvs-kernel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/07multi-client-tests target=_blank>05motr-single-node/07multi-client-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/12sns-cc-io-tests target=_blank>05motr-single-node/12sns-cc-io-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/39mount-fail target=_blank>05motr-single-node/39mount-fail</a></p></details></td></tr><tr><td>:heavy_check_mark:Passed</td><td>57</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/00userspace-tests target=_blank>01motr-single-node/00userspace-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/01kernel-tests target=_blank>01motr-single-node/01kernel-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/03initscript-tests target=_blank>01motr-single-node/03initscript-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/04net-tests target=_blank>01motr-single-node/04net-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/05rpcping-tests target=_blank>01motr-single-node/05rpcping-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/06console-tests target=_blank>01motr-single-node/06console-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/28motr-sys-kvs-test target=_blank>01motr-single-node/28motr-sys-kvs-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/29confgen-test target=_blank>01motr-single-node/29confgen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/30hagen-test target=_blank>01motr-single-node/30hagen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/35m0singlenode target=_blank>01motr-single-node/35m0singlenode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/46m0crate target=_blank>01motr-single-node/46m0crate</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/01motr-single-node/52singlenode-sanity target=_blank>01motr-single-node/52singlenode-sanity</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/02system-tests target=_blank>02motr-single-node/02system-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/08degraded-mode-tests target=_blank>02motr-single-node/08degraded-mode-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/09poolmach-tests target=_blank>02motr-single-node/09poolmach-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/20pool-version-test target=_blank>02motr-single-node/20pool-version-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/21spiel-test target=_blank>02motr-single-node/21spiel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/22rpc-cancel-test target=_blank>02motr-single-node/22rpc-cancel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/23m0d-signal-test target=_blank>02motr-single-node/23m0d-signal-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/24m0d-fsync-test target=_blank>02motr-single-node/24m0d-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/41motr-conf-update target=_blank>02motr-single-node/41motr-conf-update</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/47motr-tests-user-kernel target=_blank>02motr-single-node/47motr-tests-user-kernel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/02motr-single-node/53clusterusage-alert target=_blank>02motr-single-node/53clusterusage-alert</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/16conf-st-test target=_blank>03motr-single-node/16conf-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/17sss-st-test target=_blank>03motr-single-node/17sss-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/36spare-reservation target=_blank>03motr-single-node/36spare-reservation</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/54sns-repair-motr-1f target=_blank>03motr-single-node/54sns-repair-motr-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/55sns-repair-motr-1n-1f target=_blank>03motr-single-node/55sns-repair-motr-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/56sns-repair-motr-mf target=_blank>03motr-single-node/56sns-repair-motr-mf</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/57sns-repair-motr-1k-1f target=_blank>03motr-single-node/57sns-repair-motr-1k-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/58sns-repair-motr-ios-fail target=_blank>03motr-single-node/58sns-repair-motr-ios-fail</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/59sns-repair-motr-abort target=_blank>03motr-single-node/59sns-repair-motr-abort</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/60sns-repair-motr-abort-quiesce target=_blank>03motr-single-node/60sns-repair-motr-abort-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/03motr-single-node/61spiel-multi-confd target=_blank>03motr-single-node/61spiel-multi-confd</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/19sns-abort-test target=_blank>04motr-single-node/19sns-abort-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/25spiel-sns-repair-test target=_blank>04motr-single-node/25spiel-sns-repair-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/26spiel-sns-repair-quiesce-test target=_blank>04motr-single-node/26spiel-sns-repair-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/27sns-repair-io-fail-test target=_blank>04motr-single-node/27sns-repair-io-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/32m0t1fs-rconfc-fail-test target=_blank>04motr-single-node/32m0t1fs-rconfc-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/33motr-st target=_blank>04motr-single-node/33motr-st</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/34sns-repair-1n-1f target=_blank>04motr-single-node/34sns-repair-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/48motr-raid0-io target=_blank>04motr-single-node/48motr-raid0-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/50motr-rm-lock-cc-io target=_blank>04motr-single-node/50motr-rm-lock-cc-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/04motr-single-node/51motr-rmw target=_blank>04motr-single-node/51motr-rmw</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/10sns-single-tests target=_blank>05motr-single-node/10sns-single-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/11sns-multi-tests target=_blank>05motr-single-node/11sns-multi-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/13sns-repair-quiesce target=_blank>05motr-single-node/13sns-repair-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/14m0t1fs-fsync-test target=_blank>05motr-single-node/14m0t1fs-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/15m0t1fs-fwait-test target=_blank>05motr-single-node/15m0t1fs-fwait-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/18m0mount-test target=_blank>05motr-single-node/18m0mount-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/38sns-abort-quiesce-test target=_blank>05motr-single-node/38sns-abort-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/40motr-dgmode target=_blank>05motr-single-node/40motr-dgmode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/42motr-utils target=_blank>05motr-single-node/42motr-utils</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/43motr-sync-replication target=_blank>05motr-single-node/43motr-sync-replication</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/44motr-sns-repair target=_blank>05motr-single-node/44motr-sns-repair</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/05motr-single-node/45motr-sns-repair-N-1 target=_blank>05motr-single-node/45motr-sns-repair-N-1</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit/motr-single-node/62dix-repair-st-lookup-insert target=_blank>motr-single-node/62dix-repair-st-lookup-insert</a></p></details></td></tr><tr><td>Total</td><td>64</td><td><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/646//testReport/junit>:link:</a></td></tr></table><h2> CppCheck Summary</h2><p> &emsp;&emsp;&emsp;Cppcheck: No new warnings found :thumbsup:</p>

huanghua78 commented:
I have addressed all the review comments from the underlying PRs here in this PR: https://github.com/Seagate/cortx-motr/pull/883/commits/7f160f728f01eee8c5ba846828d32f55665774aa
Corresponding PRs will not be updated. I will add a comment there.

shailesh-vaidya commented:
<h1>Jenkins CI Result : <a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647/>Motr#647</a></h1><h2>Motr Test Summary</h2><table><tr><th>Test Result</th><th>Count</th><th>Info</tr><tr><td>:x:Failed</td><td>2</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/32m0t1fs-rconfc-fail-test target=_blank>04motr-single-node/32m0t1fs-rconfc-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/51kem target=_blank>02motr-single-node/51kem</a></p></details></td></tr><tr><td>:checkered_flag:Skipped</td><td>5</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/37protocol target=_blank>01motr-single-node/37protocol</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/31motr-sys-kvs-kernel-test target=_blank>04motr-single-node/31motr-sys-kvs-kernel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/07multi-client-tests target=_blank>05motr-single-node/07multi-client-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/12sns-cc-io-tests target=_blank>05motr-single-node/12sns-cc-io-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/39mount-fail target=_blank>05motr-single-node/39mount-fail</a></p></details></td></tr><tr><td>:heavy_check_mark:Passed</td><td>57</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/00userspace-tests target=_blank>01motr-single-node/00userspace-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/01kernel-tests target=_blank>01motr-single-node/01kernel-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/03initscript-tests target=_blank>01motr-single-node/03initscript-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/04net-tests target=_blank>01motr-single-node/04net-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/05rpcping-tests target=_blank>01motr-single-node/05rpcping-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/06console-tests target=_blank>01motr-single-node/06console-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/28motr-sys-kvs-test target=_blank>01motr-single-node/28motr-sys-kvs-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/29confgen-test target=_blank>01motr-single-node/29confgen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/30hagen-test target=_blank>01motr-single-node/30hagen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/35m0singlenode target=_blank>01motr-single-node/35m0singlenode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/46m0crate target=_blank>01motr-single-node/46m0crate</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/01motr-single-node/52singlenode-sanity target=_blank>01motr-single-node/52singlenode-sanity</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/02system-tests target=_blank>02motr-single-node/02system-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/08degraded-mode-tests target=_blank>02motr-single-node/08degraded-mode-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/09poolmach-tests target=_blank>02motr-single-node/09poolmach-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/20pool-version-test target=_blank>02motr-single-node/20pool-version-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/21spiel-test target=_blank>02motr-single-node/21spiel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/22rpc-cancel-test target=_blank>02motr-single-node/22rpc-cancel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/23m0d-signal-test target=_blank>02motr-single-node/23m0d-signal-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/24m0d-fsync-test target=_blank>02motr-single-node/24m0d-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/41motr-conf-update target=_blank>02motr-single-node/41motr-conf-update</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/47motr-tests-user-kernel target=_blank>02motr-single-node/47motr-tests-user-kernel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/02motr-single-node/53clusterusage-alert target=_blank>02motr-single-node/53clusterusage-alert</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/16conf-st-test target=_blank>03motr-single-node/16conf-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/17sss-st-test target=_blank>03motr-single-node/17sss-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/36spare-reservation target=_blank>03motr-single-node/36spare-reservation</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/54sns-repair-motr-1f target=_blank>03motr-single-node/54sns-repair-motr-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/55sns-repair-motr-1n-1f target=_blank>03motr-single-node/55sns-repair-motr-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/56sns-repair-motr-mf target=_blank>03motr-single-node/56sns-repair-motr-mf</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/57sns-repair-motr-1k-1f target=_blank>03motr-single-node/57sns-repair-motr-1k-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/58sns-repair-motr-ios-fail target=_blank>03motr-single-node/58sns-repair-motr-ios-fail</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/59sns-repair-motr-abort target=_blank>03motr-single-node/59sns-repair-motr-abort</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/60sns-repair-motr-abort-quiesce target=_blank>03motr-single-node/60sns-repair-motr-abort-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/03motr-single-node/61spiel-multi-confd target=_blank>03motr-single-node/61spiel-multi-confd</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/19sns-abort-test target=_blank>04motr-single-node/19sns-abort-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/25spiel-sns-repair-test target=_blank>04motr-single-node/25spiel-sns-repair-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/26spiel-sns-repair-quiesce-test target=_blank>04motr-single-node/26spiel-sns-repair-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/27sns-repair-io-fail-test target=_blank>04motr-single-node/27sns-repair-io-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/33motr-st target=_blank>04motr-single-node/33motr-st</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/34sns-repair-1n-1f target=_blank>04motr-single-node/34sns-repair-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/48motr-raid0-io target=_blank>04motr-single-node/48motr-raid0-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/49motr-rpc-cancel target=_blank>04motr-single-node/49motr-rpc-cancel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/50motr-rm-lock-cc-io target=_blank>04motr-single-node/50motr-rm-lock-cc-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/04motr-single-node/51motr-rmw target=_blank>04motr-single-node/51motr-rmw</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/10sns-single-tests target=_blank>05motr-single-node/10sns-single-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/11sns-multi-tests target=_blank>05motr-single-node/11sns-multi-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/13sns-repair-quiesce target=_blank>05motr-single-node/13sns-repair-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/14m0t1fs-fsync-test target=_blank>05motr-single-node/14m0t1fs-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/15m0t1fs-fwait-test target=_blank>05motr-single-node/15m0t1fs-fwait-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/18m0mount-test target=_blank>05motr-single-node/18m0mount-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/38sns-abort-quiesce-test target=_blank>05motr-single-node/38sns-abort-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/40motr-dgmode target=_blank>05motr-single-node/40motr-dgmode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/42motr-utils target=_blank>05motr-single-node/42motr-utils</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/43motr-sync-replication target=_blank>05motr-single-node/43motr-sync-replication</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/44motr-sns-repair target=_blank>05motr-single-node/44motr-sns-repair</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/05motr-single-node/45motr-sns-repair-N-1 target=_blank>05motr-single-node/45motr-sns-repair-N-1</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit/motr-single-node/62dix-repair-st-lookup-insert target=_blank>motr-single-node/62dix-repair-st-lookup-insert</a></p></details></td></tr><tr><td>Total</td><td>64</td><td><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/647//testReport/junit>:link:</a></td></tr></table><h2> CppCheck Summary</h2><p> &emsp;&emsp;&emsp;Cppcheck: No new warnings found :thumbsup:</p>

huanghua78 commented:
All the PRs included by this PR have been reviewed and approved.
They are integrated and tested here.

I am going to merge them into fdmi_int branch in #719.
